### ISSUE TITLE
Empty text files

### ISSUE THREAD

**Issue by ravidreams** [tshrinivasan/OCR4wikisource#20]
Shrini, 

Regarding your comment at 

https://github.com/tshrinivasan/OCR4wikisource/issues/10#issuecomment-168553261

//all the 92 pages uploaded fine and got the 92 text files.//

Can you check if there is text inside those text files?

I had all empty page files and text files. 

I checked manual upload of single page pdf to Google Drive. In google drive itself it is empty. 

We need to find if this is a problem with the way we do scanning / making pdf files and recommend the right process.

**Comment by tshrinivasan** [comment]
Is the empty files are only for 
 https://upload.wikimedia.org/wikipedia/commons/7/7e/Indradhanu%2C_Akhi_O_Kabitar_Dirgha_JIban.pdf

I too get empty pages only.

Do you get content for other PDF files?

**Comment by ravidreams** [comment]
Yes, only for https://upload.wikimedia.org/wikipedia/commons/7/7e/Indradhanu%2C_Akhi_O_Kabitar_Dirgha_JIban.pdf

Problem may not be with the script. My guess is it should be with scanning / making pdf files. Let us leave this thread open to report future instances of such files. It may help us identify a pattern.
